### ISSUE TITLE
AttributeError with missing email in authors entry

### ISSUE THREAD

**Issue by blueyed** [sdispater/poetry#66]
When using no email with entries in `authors` (`authors = ["Alice"]`),
`poetry -vvv build` fails:
                                              
```
  [AttributeError]                            
  'NoneType' object has no attribute 'group'  
                                              

Exception trace:
 /home/user/.local/lib/python3.6/site-packages/cleo/application.py in run() at line 94
   status_code = self.do_run(input_, output_)
 /home/user/.local/lib/python3.6/site-packages/poetry/console/application.py in do_run() at line 74
   return super(Application, self).do_run(i, o)
 /home/user/.local/lib/python3.6/site-packages/cleo/application.py in do_run() at line 197
   status_code = command.run(input_, output_)
 /home/user/.local/lib/python3.6/site-packages/poetry/console/commands/command.py in run() at line 22
   return super(BaseCommand, self).run(i, o)
 /home/user/.local/lib/python3.6/site-packages/cleo/commands/base_command.py in run() at line 146
   status_code = self.execute(input_, output_)
 /home/user/.local/lib/python3.6/site-packages/cleo/commands/command.py in execute() at line 107
   return self.handle()
 /home/user/.local/lib/python3.6/site-packages/poetry/console/commands/build.py in handle() at line 26
   builder.build(fmt)
 /home/user/.local/lib/python3.6/site-packages/poetry/masonry/builder.py in build() at line 23
   builder = self._FORMATS[fmt](self._poetry, self._venv, self._io)
 /home/user/.local/lib/python3.6/site-packages/poetry/masonry/builders/builder.py in __init__() at line 41
   self._meta = Metadata.from_package(self._package)
 /home/user/.local/lib/python3.6/site-packages/poetry/masonry/metadata.py in from_package() at line 53
   meta.author = package.author_name
 /home/user/.local/lib/python3.6/site-packages/poetry/packages/package.py in author_name() at line 153
   return self._get_author()['name']
 /home/user/.local/lib/python3.6/site-packages/poetry/packages/package.py in _get_author() at line 168
   name = m.group('name')
```

I think the email should be allowed to be empty, which means that this should not crash.

**Comment by sdispater** [comment]
@blueyed Commit 5b14bd1 in the `develop` makes the email address optional. It will be available in the next `0.9.0` release.

**Comment by sdispater** [comment]
Release `0.9.0` is out!
